### PR TITLE
Add autoclose guidance

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Adding updates from https://github.com/blackbaud/skyux2-docs/pull/1003/files that address https://github.com/blackbaud/skyux-toast/issues/53 so they don't get lost when we switch over to the new version of the docs.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720](https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720)

File: src/app/docs/toast/toast-docs.component.html [hunk 1]
```diff
@@ -41,6 +41,9 @@
         <p>
           The text must be clear and concise, and punctuation is subjective depending on the scenario.
         </p>
+        <p>
+          If a toast is the only place users can see a message, display the toast until users manually close it. Only close toasts automatically if users can access the messages after the toasts close.
```
Comment (Blackbaud-ToddRoberts) [inline src/app/docs/toast/toast-docs.component.html]: This should be in the Behavior section rather than Content

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720](https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720](https://developer.blackbaud.com/skyux-toast?_sp.spa=pr-toast-close-message-082720)

**Pull request merged**